pypeit/pypeitsetup.py

```diff
@@ -419,22 +431,27 @@ def run(self, setup_only=False, calibration_check=False,
         if self.fitstbl is None:
             self.build_fitstbl(strict=not setup_only)#, bkg_pairs=bkg_pairs)
 
+        # Remove frames that have invalid values for
+        # configuration-defining metadata
+        if clean_config:
+            self.fitstbl.clean_configurations()
+
         # File typing
         self.get_frame_types(flag_unknown=setup_only or calibration_check,
                              use_header_id=use_header_id)
 
         # Determine the configurations and assign each frame to the
         # specified configuration
-        ignore_frames=['bias', 'dark']
-        cfgs = self.fitstbl.unique_configurations(ignore_frames=ignore_frames)
-        self.fitstbl.set_configurations(cfgs, ignore_frames=ignore_frames)
+        cfgs = self.fitstbl.unique_configurations()
+        self.fitstbl.set_configurations(cfgs)
 
         # Assign frames to calibration groups
-        self.fitstbl.set_calibration_groups(global_frames=['bias', 'dark'])
+        self.fitstbl.set_calibration_groups() #global_frames=['bias', 'dark'])
 
         # Set default comb_id
         self.fitstbl.set_combination_groups()
 
+        # TODO: Are we planning to do this?
```

Comment:
I have to admit I can't remember this..

Reply:
This might have actually been a place-holder that I put in.  I'll just leave it for now.